public/apps/configuration/utils/role-list-utils.tsx

```diff
@@ -92,19 +92,24 @@ export function buildSearchFilterOptions(roleList: any[], attrName: string): Arr
 }
 
 // Submit request to delete given roles. No error handling in this function.
-export async function requestDeleteRoles(http: HttpStart, roles: string[]) {
+export async function requestDeleteRoles(http: HttpStart, roles: string[], query?: HttpFetchQuery) {
```

Comment:
Can we make this required? Any other usages?

Reply:
sure.